site/content/en/docs/Installation/Terraform/oke.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Installing Agones on OCI Kubernetes Engine using Terraform"
+linkTitle: "OCI"
+weight: 20
+publishDate: 2024-10-21
+description: >
+  You can use Terraform to provision an OKE cluster and install Agones on it.
+---
+
+## Installation
+
+You can use Terraform to provision your OKE (OCI Kubernetes Engine) cluster and install Agones on it using the Helm Terraform provider.
```

Comment:
```suggestion
You can use Terraform to provision your OKE (Oracle Kubernetes Engine) cluster and install Agones on it using the Helm Terraform provider.
```

I just wanted to have "Oracle" on the page for those people who may not know, or haven't read the other OCI page. Does this still make sense?